app/helpers/actionTicketManager/__tests__/index_spec.tsx

```diff
@@ -0,0 +1,93 @@
+import * as store from "store";
+import ActionTicketManager, { DEVICE_ID_KEY, SESSION_ID_KEY, Ticket, ActionTicketParams, TICKET_QUEUE_KEY } from "..";
+
+describe("ActionTicketManager helper", () => {
+  const mockTicketParams: ActionTicketParams = {
+    pageUrl: "https://scinapse.io",
+    actionTarget: "query",
+    actionType: "fire",
+    actionTag: "Drosophila",
+  };
+
+  describe("trackTicket Method", () => {
+    let ticket: Ticket;
+    let deviceKey: string;
+    let sessionKey: string;
+
+    beforeAll(() => {
+      ActionTicketManager.trackTicket(mockTicketParams);
+    });
+
+    afterAll(() => {
+      store.clearAll();
+    });
+
+    beforeEach(() => {
+      ticket = store.get(TICKET_QUEUE_KEY)[0];
+      deviceKey = store.get(DEVICE_ID_KEY);
+      sessionKey = store.get(SESSION_ID_KEY);
+    });
+
+    it("should add ticket to ticket queue", () => {
+      expect(ticket).toMatchObject(mockTicketParams);
+    });
+
+    it("should return proper device id", () => {
+      expect(ticket.deviceId).toEqual(deviceKey);
+    });
+
+    it("should return proper session id", () => {
+      expect(ticket.sessionId).toEqual(sessionKey);
+    });
+
+    it("should return proper createdAt attribute format(ISO-8601 with time zone)", () => {
+      expect(ticket.createdAt).toMatch(
+        // From https://stackoverflow.com/questions/3143070/javascript-regex-iso-datetime
+        // tslint:disable-next-line:max-line-length
+        /(\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d\.\d+([+-][0-2]\d:[0-5]\d|Z))|(\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d([+-][0-2]\d:[0-5]\d|Z))|(\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d([+-][0-2]\d:[0-5]\d|Z))/
```

Comment:
It looks only the first alternative (`(\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d\.\d+([+-][0-2]\d:[0-5]\d|Z))`) is needed for format checking.